test/serializer-test.ts

```diff
@@ -55,28 +64,47 @@ function makeTransaction(
   const currencyAmount = new CurrencyAmount()
   currencyAmount.setXrpAmount(paymentAmount)
 
-  const destinationAddress = new AccountAddress()
-  destinationAddress.setAddress(destination)
+  const amount = new Amount()
+  amount.setValue(currencyAmount)
+
+  const destinationAccountAddress = new AccountAddress()
+  destinationAccountAddress.setAddress(destinationAddress)
+
+  const destination = new Destination()
+  destination.setValue(destinationAccountAddress)
 
   const payment = new Payment()
-  payment.setDestination(destinationAddress)
-  payment.setAmount(currencyAmount)
+  payment.setDestination(destination)
```

Comment:
This now takes a `Destination` instead of a `AccountAddress`? What is a `Destination`?

Reply:
A destination is just a wrapper for an account address which is specifically for a destination address. 

It's just another layer of abstraction introduced to make it easy to work with the rpc code on the node.